portfolio/src/main/client/src/components/controls/toast.ts

```diff
@@ -0,0 +1,166 @@
+import {htmlElement} from '@src/util/html';
+import '@res/style/controls/toast.scss';
+
+type ToastKind = 'info' | 'error';
+
+interface ToastInfo {
+  message: string;
+  kind: 'info' | 'error';
+
+  timeout: number | false;
```

Comment:
nit: use `null` or `undefined` instead of `false` as your _no-op_ value. `undefined` if it should be disabled by default, and `null` if you want to have a default value that users have to explicitly disable,